### DISCUSSION TITLE
Set height for a Carousel

### DISCUSSION THREAD

**Discussion by DrakoPOD** [quasarframework/quasar#9506] [General - Components / Directives / etc]
I tried many things, but I can't get a carousel with a set height, if the images have different height, the scroll bar start blink constantly when changing between images, generating an ugly effect in my page:
https://codepen.io/drakopod/pen/bGqYLya

The carousel is inside of a qDialog, have a col class. Also, due by the Carousel doesn't allow me to fit images, I created `carousel-slide` with an `img` component. The thumbnails are buttons with img.

**Comment by metalsadman** [comment]
@DrakoPOD set the styles on the q-img itself ie. `<q-img style="height: 200px; width: 200px" ...`.
